### DISCUSSION TITLE
mdBook support for vercel

### DISCUSSION THREAD

**Discussion by tsathis** [vercel/community#3322] [Help] [answered]
[mdBook](https://rust-lang.github.io/mdBook/) is another great tool to create static websites using markdown sources. It would be great if we could directly deploy mdBooks in Vercel. In other words, I would like to see 'mdBook' in the following 'Framework Preset' list.

![image](https://user-images.githubusercontent.com/34299693/175765612-e30738bf-b5b0-4142-8564-cb7d5470cf7c.png)

**Comment by Confucian-PUT** [comment]
Agree that!

**Reply by csk111165** [reply]
Any update on the same??

**Comment by ellie** [comment, marked as answer]
Overriding the "build" command and setting it to

```
curl -Lo mdbook.tar.gz https://github.com/rust-lang/mdBook/releases/download/v0.4.32/mdbook-v0.4.32-x86_64-unknown-linux-musl.tar.gz; tar -xvzf mdbook.tar.gz; ./mdbook build
```

Works nicely! I originally had it build from source, but that's _slow_. This builds and deploys in 4s 🚀 

See: https://notes.ellie.wtf

**Reply by flossypurse** [reply]
Fantastic.
